app.js

```diff
@@ -0,0 +1,25 @@
+// express modules
+const express = require('express');
+
+
+// third party libararies
+const open = require('open');
+const path = require('path');
+const compression = require('compression');
+
+const app = express();
+
+const port = process.env.PORT || 8001;
+
+app.use(compression());
+
+// set static file directory
+app.use(express.static('build'));
+
+// catch all request at index page
+app.get('*', (req, res) => {
+  res.sendFile(path.join(__dirname, 'build/index.html'));
+});
+
+// bind port
+app.listen(port, () => process.env.PORT === 8001 && open(`http://localhost:${port}`));
```

Comment:
#### Critical
We talked about this. Why do you need to check the port again when it's declared above?